dts/bindings/kscan/hynitron-cst816s.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 Qingsong Gou <gouqs@hotmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Hynitron CST816S touchscreen sensor
+
+compatible: "hynitron-cst816s"
```

Comment:
I think this deserves a real vendor prefix now that you are adding a driver.

Please change this PR to do this instead:

1. change this to `hynitron,cst816s`
2. rename the file to `hynitron,cst816s.yaml`
3. add the line `hynitron	Hynitron` to dts/bindings/vendor-prefixes.txt in the right place?
4. update boards/arm/pinetime_devkit0/pinetime_devkit0.dts to use the new compatible

Please check 1543d2e0a7ebadc6b4e73cc9ad28a73fd03537b3, where I wrote:

```
    We can revisit this if anyone wants to upstream drivers for these
    compatibles in the future, but for now it doesn't seem worth it.
```

Now that you are adding a driver, it is OK to add a vendor prefix, I think.